src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -457,6 +457,52 @@ public String toString() {
     }
   };
 
+  /**
+   * Create a UserACL object from the ACL GETUSER < > result
+   */
+  public static final Builder<UserACL> USER_INFO = new Builder<UserACL>() {
```

Comment:
Please use variable same as class name (may with underscore).

Reply:
I have solved that in the commit 439d7aa692cd0ea4116d6307eefac2a76f5ca5f0

Reply:
I have renamed the class to `AccessControlUser`, and use the `ACCESS_CONTROL_USER` as the variable name.